index.html

```diff
@@ -47,35 +47,57 @@ <h2 class='player3-score'>player 3 score</h2>
+      <div class='puzzle-grid-square tile-2'></div>
+      <div class='puzzle-grid-square tile-3'></div>
+      <div class='puzzle-grid-square tile-4'></div>
+      <div class='puzzle-grid-square tile-5'></div>
+      <div class='puzzle-grid-square tile-6'></div>
+      <div class='puzzle-grid-square tile-7'></div>
+      <div class='puzzle-grid-square tile-8'></div>
+      <div class='puzzle-grid-square tile-9'></div>
+      <div class='puzzle-grid-square tile-10'></div>
+      <div class='puzzle-grid-square tile-11'></div>
+      <div class='puzzle-grid-square tile-12'></div>
+      <div class='puzzle-grid-square tile-13'></div>
+      <div class='puzzle-grid-square top-right-tile green-tile'></div>
+      <div class='puzzle-grid-square top-left-tile green-tile'></div>
+      <div class='puzzle-grid-square tile-14'></div>
+      <div class='puzzle-grid-square tile-15'></div>
+      <div class='puzzle-grid-square tile-16'></div>
+      <div class='puzzle-grid-square tile-17'></div>
+      <div class='puzzle-grid-square tile-18'></div>
+      <div class='puzzle-grid-square tile-19'></div>
+      <div class='puzzle-grid-square tile-20'></div>
+      <div class='puzzle-grid-square tile-21'></div>
+      <div class='puzzle-grid-square tile-22'></div>
+      <div class='puzzle-grid-square tile-23'></div>
+      <div class='puzzle-grid-square tile-24'></div>
+      <div class='puzzle-grid-square tile-25'></div>
+      <div class='puzzle-grid-square tile-26'></div>
+      <div class='puzzle-grid-square tile-27'></div>
       <div class='puzzle-grid-square top-right-tile green-tile'></div>
       <div class='puzzle-grid-square bottom-left-tile green-tile'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
-      <div class='puzzle-grid-square'></div>
+      <div class='puzzle-grid-square tile-28'></div>
+      <div class='puzzle-grid-square tile-29'></div>
+      <div class='puzzle-grid-square tile-30'></div>
+      <div class='puzzle-grid-square tile-31'></div>
+      <div class='puzzle-grid-square tile-32'></div>
+      <div class='puzzle-grid-square tile-33'></div>
+      <div class='puzzle-grid-square tile-34'></div>
+      <div class='puzzle-grid-square tile-35'></div>
+      <div class='puzzle-grid-square tile-36'></div>
+      <div class='puzzle-grid-square tile-37'></div>
+      <div class='puzzle-grid-square tile-38'></div>
+      <div class='puzzle-grid-square tile-39'></div>
+      <div class='puzzle-grid-square tile-40'></div>
+      <div class='puzzle-grid-square tile-41'></div>
```

Comment:
There are a lot of tiles to account for in your HTML. Have you thought about possibly dynamically generating these with JS? This would eliminate the need to hook into the class names to set the letters.